packages/react-i18n/src/i18n.ts

```diff
@@ -170,6 +152,18 @@ export default class I18n {
       ...options,
     }).format(date);
   }
+
+  weekStartDay(argCountry?: I18n['defaultCountry']) {
+    const country = argCountry || this.defaultCountry;
+
+    if (!country) {
+      throw new MissingTimezoneError(
```

Comment:
You'll want to create an error just for this, the error class is misleading.

Reply:
🤦‍♂️ 
I read it as regular `Error`. I'll change that.